benchmarks/maxtext_xpk_runner.py

```diff
@@ -257,21 +269,25 @@ def run_command_with_updates(command, task, verbose=True) -> int:
 
 
 def build_user_command(
```

Comment:
currently it's adding extra pathway configs by default, we can either add maxtext_xpk_pw_runner.py or another build_user_commands function. what do you think? @SujeethJinesh

Reply:
The `PathwaysConfig` is only used if `--use_pathways` is passed in. This is intentional because it would make it easier for us to have similar configs and code changes as McJAX but just make drop in replacements to support pathways. I think having a new function or file would bifurcate the code too much and make it harder to share configs if needed over time.